sajibportfolio.html

```diff
@@ -1,289 +1,294 @@
+            <div class="jt">
+              <p class="jtt">Back End Dev</p>
+            </div>
+            <span class="ct1"></span>
+            <!--<img class="counter" src="statics/images/Counter.png" alt="counter">-->
+            <div class="yr">
+              <p class="yrt">2015</p>
             </div>
           </div>
-          <p class="jhora">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <ul class="tgs">
-            <li class="li1">
-              <p class="lp1">html </p>
-            </li>
-            <li class="li2">
-              <p class="lp2">css</p>
-            </li>
-            <li class="li3">
-              <p class="lp3">javascript</p>
-            </li>
-          </ul>
-          <div class="btl">
-            <button class="btn"><a href="#" class="btt">See Project</a></button>
-          </div>
-  
         </div>
-  
-      </section>
-      <section class="mwp3">
-        <div class="sp3"></div>
-        <div class="lb2">
-          <div class='pt2'>
-            <h2 class="prt2">Tonic </h2>
-            <div class="fr2">
-              <div class="tl">
-                <p class="tlt">CANOPY</p>
-              </div>
-              <span class="ct"></span>
-              <!-- <img class="counter" src="statics/images/Counter.png" alt="counter"/> -->
-              <div class="jt">
-                <p class="jtt">Back End Dev</p>
-              </div>
-              <span class="ct1"></span>
-              <!--<img class="counter" src="statics/images/Counter.png" alt="counter">-->
-              <div class="yr">
-                <p class="yrt">2015</p>
-              </div>
+        <p class="jhora">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tgs">
+          <li class="li1">
+            <p class="lp1">html </p>
+          </li>
+          <li class="li2">
+            <p class="lp2">css</p>
+          </li>
+          <li class="li3">
+            <p class="lp3">javascript</p>
+          </li>
+        </ul>
+        <div class="btl">
+          <button class="btn"><a href="#" class="btt">See Project</a></button>
+        </div>
+
+      </div>
+
+    </section>
+    <section class="mwp3">
+      <div class="sp3"></div>
+      <div class="lb2">
+        <div class='pt2'>
+          <h2 class="prt2">Tonic </h2>
+          <div class="fr2">
+            <div class="tl">
+              <p class="tlt">CANOPY</p>
+            </div>
+            <span class="ct"></span>
+            <!-- <img class="counter" src="statics/images/Counter.png" alt="counter"/> -->
+            <div class="jt">
+              <p class="jtt">Back End Dev</p>
+            </div>
+            <span class="ct1"></span>
+            <!--<img class="counter" src="statics/images/Counter.png" alt="counter">-->
+            <div class="yr">
+              <p class="yrt">2015</p>
             </div>
           </div>
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure the project card follows the figma template as close as possible, also your 3rd and 4th project cards does not have the images displayed see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206856412-0649032b-d094-4f39-8051-f4389c2796c0.png)

currently you have excessive hap between project images and project details, how It is :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206856453-1e3270c5-c18e-44a8-9b11-858c351b0e6f.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206856501-542a5497-405a-4705-ad08-cf38d05c71db.png)

you have morespace inbetween and no space on left and right side.

Reply:
removed the space in between thanks.

Reply:
removed comments from html

Reply:
fixed changing the image url which was relative for card 3 and 4